karapace/schema_registry_apis.py

```diff
@@ -511,6 +512,9 @@ def _has_subject_with_id() -> bool:
             response_body["references"] = [r.to_dict() for r in schema.references]
         if fetch_max_id:
             response_body["maxId"] = schema.max_id
+        if include_subjects:
+            response_body["subjects"] = self.schema_registry.database.subjects_for_schema(parsed_schema_id)
```

Comment:
I suspect this is quite cheap to include, I would just always include it.

Including fields conditionally makes it very hard to expose a strict schema once we start introducing FastAPI (and OpenAPI).

Reply:
Do we have a valid load test to check the difference?